code/handlers/trackservers.q

```diff
@@ -137,8 +135,8 @@ addh:{[hpuP]
     addhw[hpuP;W]]}
 
 // return the details of the current process
-getdetails:{(.z.f;.z.h;system"p";@[value;`.proc.procname;`];@[value;`.proc.proctype;`];@[value;(`.proc.getattributes;`);()!()])}
-
+getdetails:{(.z.f;.z.h;system"p";@[value;`.proc.procname;`];@[value;`.proc.proctype;`];@[value;(`.proc.ges;`);()!()])}
+//getdetails:{(.z.f;.z.h;system"p";@[value;`.proc.procname;`];@[value;`.proc.proctype;`])}
```

Comment:
Remove comment.

Reply:
That was not one of my changes, but I will remove it.

Reply:
You added it in this commit https://github.com/AquaQAnalytics/TorQ/commit/7f48c88246a459581982241b82dc46027633b9b3

Reply:
Yes remove commented out getdetails definition